src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -641,7 +641,7 @@ trait BeamHelper extends LazyLogging {
     val src = beamConfig.beam.exchange.scenario.source.toLowerCase
     if (src == "urbansim") {
       buildUrbansimScenarioSource(injector, beamConfig)
-    } else if (src == "matsim") {
+    } else if (src == "BeamCsv") {
```

Comment:
Comparison lowercased with `BeamCsv` which is not lowercased

Reply:
good catch!